### PR TITLE
Bugfixes

### LINKED ISSUE #356: LJ Recip not included in Gromacs Energy Comparison
When comparing energies converting systems in GROMACS to LAMMPS we found that the 'LJ Recip.' component was not being included. 
- In the 'Not comparable energies' section of the output it is not listed and not included in the 'vdw total' term for GROMACS. 
- In the 'Comparable energy terms' it is not included in the nonbonded energy for GROMACS. 
- The potential energy is correct relative to GROMACS and therefore:
_nonbonded + bonded ≠ potential_

Actual simulations were run by @mrshirts , but I can provide outputs from InterMol, GROMACS, and LAMMPS to further clarify the issue if required.

### LINKED ISSUE #357: Tilt factors ignored when converting from GROMACS to LAMMPS
If tilt factors are present in the .gro file they do not show up when converting the system to LAMMPS.

### PR DESCRIPTION
**Pull request by mrshirts**
Fixes bugs where
- tilt vectors for LAMMPS were not printed (or read)  (Fixes #357)
- LJ Recip. was not added to the energy for GROMACS. (Fixes #356)

Also adds a LAMMPS test for the box vector

### CODE REVIEW

**Comment by mrshirts** [comment]
Added bugfix for OPLS formatted dihedrals.

**Comment by mrshirts** [comment]
OK, that desmond issue was the problem. 

Review if you have a chance, Christoph, otherwise I'll approve in myself in a couple of days.

File: intermol/lammps/lammps_parser.py [hunk 1]
```diff
@@ -608,12 +610,23 @@ def parse_box(self, line, dim):
...
+        if dim == 'tilt':
+            fields = [(float(field) * self.DIST) for field in line[:3]]
+            dirs = line[3:]
+            for f,d in zip(fields,dirs):
+                if d == 'xy':
+                    self.system.box_vector[1,0] = f
+                elif d == 'xz':
+                    self.system.box_vector[2,0] = f
+                elif d == 'yz':
+                    self.system.box_vector[2,1] = f
```
Comment (ctk3b) [inline intermol/lammps/lammps_parser.py]: nit: could we use more descriptive variable names?

File: intermol/desmond/desmond_parser.py [hunk 2]
```diff
@@ -772,28 +793,28 @@ def parse_constraints(self, type, current_molecule_type):
         for j in range(ff_number):
             # water constraints actually get written to rigidwater (i.e. settles) constraints.
 
-            if 'HOH' in entry_values[j] or 'AH' in entry_values[j]:
-                split = entry_values[j].split()
+            if 'HOH' in ev[j] or 'AH' in ev[j]:
```
Comment (ctk3b) [inline intermol/desmond/desmond_parser.py]: nit: I would keep the slightly more descriptive `entry_values` - already fairly tricky to deduce what these variables represent

**Pull request merged**